src/styles/utils/_variables.scss

```
@@ -0,0 +1,6 @@
+$bg-color-main:#fff;
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-header-nav-link-hover:#00acdc;
+$btn-color:#00acdc;
```

u can just combine that variable
```suggestion
$light-blue-color: #00acdc
```

---

Thank you!)

---

Fixed ✔